### PR TITLE
Limit release to draft only

### PR DESCRIPTION
**Pull request by yongtang**
A separate step is to push the button of draft to publish it.

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

### CODE REVIEW

**Review by chrisohaver — CHANGES_REQUESTED** [review]
If the reason for this change is to defer the release trigger action until after the uploads are complete, it should update the release to be not a draft after the uploads are done.

File: Makefile.release [hunk 1]
```diff
@@ -108,6 +108,10 @@ else
 	      --data-binary "@$$asset" \
 	      "https://uploads.github.com/repos/$(GITHUB)/$(NAME)/releases/$(RELEASE)/assets?name=$${asset}" ; \
 	done )
+	@( curl -X PATCH \
+	    -H "Authorization: token ${GITHUB_ACCESS_TOKEN}" \
+            "https://api.github.com/repos/$(GITHUB)/$(NAME)/releases/$(RELEASE)" \
+            -d '{"drafe":"false"}' )
```
Comment (chrisohaver) [inline Makefile.release]:
```suggestion
            -d '{"draft":"false"}' )
```

**Review by chrisohaver — CHANGES_REQUESTED** [review]

**Comment by yongtang** [comment]
@chrisohaver  PR updated.

**Comment by chrisohaver** [comment]
https://github.com/coredns/coredns/pull/5146#issuecomment-1029264341

**Pull request closed without merging**